w3f-ring-proof/src/lib.rs

```diff
@@ -160,7 +138,8 @@ mod tests {
         let domain = Domain::new(domain_size, true);
         let h = EdwardsAffine::rand(rng);
         let seed = EdwardsAffine::rand(rng);
-        let piop_params = PiopParams::setup(domain, h, seed);
+        let pad = EdwardsAffine::rand(rng);
```

Comment:
I assume it is using this https://github.com/arkworks-rs/algebra/blob/master/ec/src/models/twisted_edwards/affine.rs#L279 and it ends up in the prime order subgroup group.

Reply:
Once I followed the execution trace and yes it lands there